### PR TITLE
REF: Parametrize value_counts tests

### PR DESCRIPTION
**Pull request by dsaxton**
Parametrizes the for loop in `test_series_groupby_value_counts`.  As a side note, this test seems to run for a pretty long time (a minute and a half); should it operate on less data perhaps?

### CODE REVIEW

**Comment by TomAugspurger** [comment]
Not sure about reducing the number of cases. Looking into the history of
the test may shed some light.

**Comment by WillAyd** [comment]
Yea a minute and a half for one benchmark is too much. Is there a way to get it down to a few seconds and still get the same measurement?

File: pandas/tests/groupby/test_value_counts.py [hunk 1]
```diff
@@ -52,29 +52,28 @@ def seed_df(seed_nans, n, m):
 
 @pytest.mark.slow
 @pytest.mark.parametrize("df, keys, bins, n, m", binned, ids=ids)
-def test_series_groupby_value_counts(df, keys, bins, n, m):
+@pytest.mark.parametrize(
+    "isort, normalize, sort, ascending, dropna", list(product((False, True), repeat=5))
```
Comment (WillAyd) [inline pandas/tests/groupby/test_value_counts.py]: Not really a fan of using `itertools` to generate parametrization. Could you instead write this as:

```python
@pytest.mark.parametrize("isort", [True, False])
@pytest.mark.parametrize("normalize", [True, False])
...
```

Equivalent at the end of the day but I think clearer and match the rest of the codebase

**Review by WillAyd — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
this is a comprehensive test and is marked as slow, so perf is ok, other than @WillAyd comment lgtm.

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @dsaxton

**Pull request merged**